.github/workflows/deploy.yml

```diff
@@ -53,4 +53,9 @@ jobs:
           npx lerna publish from-package \
             --force-publish='*' \
             --yes
-      - uses: softprops/action-gh-release@de2c0eb89ae2a093876385947365aca7b0e5f844 # v0.1.15
+      - run: |
+          PKG_VERSION=$(jq -r .version < lerna.json)
+          gh release create v$PKG_VERSION \
+            --title "Release v$PKG_VERSION" \
```

Comment:
Our github releases do not use the `v` syntax for releases. Not sure we want to move to doing that now